src/fluree/db/ledger/proto.cljc

```diff
@@ -21,4 +21,5 @@
   ;; alias name for graph
   (-alias [ledger] "Returns the ledger local alias / graph name")
   (-address [ledger] "Returns the permanent ledger address")
+  (-default-context [ledger t] "Returns the default context at time t")
```

Comment:
Does this need to be a protocol method? I tend to think of protocols as a way to get polymorphism at the "type" level, but we only have one type of ledger and even if we have several I don't know that we'll have separate implementations of `default-context`.

Reply:
🤷🏻‍♂️ Everything else is and there's only one type of ledger, so it's hard for me to speculate what would or would not be appropriate when / if there are more than one type of ledger.

Reply:
The one type of ledger we currently have is called `JsonLDLedger`. But other types of ledgers (i.e. non-JSON-LD) wouldn't necessarily even have a context in the same sense, much less a default one. So in that sense, no, it doesn't belong in here.

OTOH the db protocol (whose only implementer is likewise called `JsonLdDb`) has a `-default-context` method in it, so I'm also not sure it makes sense to make this inconsistent with that.

I don't have a strong opinion either way.

Reply:
I figure we can do a comprehensive review of the protocols at some point and move things around then, but in general  I lean towards not adding methods if I can help it.